ping.go

```diff
@@ -6,7 +6,14 @@ import (
 
 type HealthModule struct{}
 
-func (m HealthModule) Register(e *gin.Engine) {
+func (m HealthModule) Register(g *gin.RouterGroup) {
+	g.GET("/ping", m.ping)
+	g.GET("/health", m.health)
+}
+
+// Deprecated because they are not part of the /api group for endpoints.
+// Tentatively planned for complete removal in v6.
```

Comment:
As Codacy/revive warns about here, doc comments needs to start with the name of the function.

And according to the godoc recommendations, the deprecation notice should include a colon:

> To signal that an identifier should not be used, add a paragraph to its doc comment that begins with “Deprecated:” followed by some information about the deprecation.
>
> https://go.dev/blog/godoc

```suggestion
// DeprecatedRegister adds API health-related endpoints to a Gin-Gonic engine.
//
// Deprecated: Not part of the /api group for endpoints. Tentatively planned
// for complete removal in v6.
```